Solutions/CyberArkEPM/DataConnectors/requirements.txt

```diff
@@ -0,0 +1,7 @@
+# DO NOT include azure-functions-worker in this file
+# The Python Worker is managed by Azure Functions platform
+# Manually managing azure-functions-worker may cause unexpected issues
+
+azure-storage-file-share==12.5.0
```

Comment:
Hey @averbn, the version for "azure-storage-file-share" is 12.5.0 , but not updated in [CyberArkEPMSentinelConn.zip ](https://github.com/socprime/Azure-Sentinel/blob/CyberArkEPM_Dataconnector/Solutions/CyberArkEPM/DataConnectors/CyberArkEPMSentinelConn.zip?raw=true), it is "azure_storage_file_share-12.3.0.dist-info", please check and update , thanks.